proto/hummock.proto

```diff
@@ -252,6 +261,7 @@ service HummockManagerService {
   rpc ReportVacuumTask(ReportVacuumTaskRequest) returns (ReportVacuumTaskResponse);
   rpc GetCompactionGroups(GetCompactionGroupsRequest) returns (GetCompactionGroupsResponse);
   rpc TriggerManualCompaction(TriggerManualCompactionRequest) returns (TriggerManualCompactionResponse);
+  rpc ListSstableIdInfos(ListSstableIdInfosRequest) returns (ListSstableIdInfosResponse);
```

Comment:
I'm thinking whether we should rename all functions solely used by risectl to like `RiseCtlListSstableIdInfos` (or move them to a separate proto file in the future).

The reason behind this is that these RPCs are only used by risectl. If used by streaming executors or normal meta client, things would go wrong -- e.g. developers need to pin a version before using it, and versions (in the future) will be pushed to clients instead of being fetched using RPC. In this case, developers might easily violate those conventions if they accidentally use a RPC designed for risectl.

Reply:
I think this is a good idea but it can be handled in a future PR

Reply:
I think separating them would be a good idea, but that job would be complicated enough that it would have to be handled in a separate PR.